operators.js

```diff
@@ -396,9 +400,9 @@ function toPullStream() {
     function paginateStream(ops) {
       let seq = meta.seq || 0
       let total = Infinity
-      const limit = meta.pageSize || 1
+      const limit = meta.pageSize || meta.batchSize || 20
```

Comment:
The default `1` was actually pretty bad for performance. Reminder: https://github.com/ssb-ngi-pointer/jitdb/pull/123#issuecomment-782734363